plugins/com.google.cloud.tools.eclipse.dataflow.ui/plugin.xml

```diff
@@ -227,21 +227,21 @@
          </separator>
          <command commandId="com.google.cloud.tools.eclipse.dataflow.ui.dataflowVersion.pin"
              label="Pin Dataflow Version"
-             tooltip="Pin the Dataflow version for this project to the current latest version">
+             tooltip="Pin the Dataflow Version for this Project to the Current Latest Version">
```

Comment:
I think we should remove these _pin/track/update_ tooltips or reword them as I don't find them helpful.  When I first started using the Dataflow plugin, I found the labels to be confusing, and the tooltips don't help.  Possible rewording:

- Pin: _Use Currently Configured Dataflow Version_ (what version is that?)
- Track: _Track Latest Available Dataflow Version_ (is it different? when is this checked?)
- Update: _Re-pin To Latest Available Dataflow Version_ (what version is that?)

But to be even more honest, I don't like these options as commands as there's no indication of whether it will do something, or to gauge the consequences.  What if 3.0.0 is now released, which makes major changes?  I'd really prefer that there be a single command that opens Dataflow properties page, and these commands be available there.